services/showcase/frontend/src/components/acme/ApplicantCard.vue

```diff
@@ -5,7 +5,11 @@
     tile
     @click="SET_SELECTED_APPLICANT(applicant)"
   >
-    <v-system-bar color="rgba(0, 160, 144)" />
+    <v-system-bar
+      v-if="applicant.verified === 'false'"
+      color="rgba(200, 060, 74)"
+    />
+    <v-system-bar v-else color="rgba(0, 160, 144)" />
```

Comment:
Doesn't really save much space but FYI this can be done with (going off memory, not sure if I have the quotes right) a directive for the color attribute

`:color="applicant.verified ? 'rgba(0, 160, 144)' : 'color="rgba(200, 060, 74)' "